Issue 3

DVEfremov opened the pull request:
change button text  on  images
add camera flip button

Commit:
A few more buttons #3

Commit:
A few more buttons #3

- fit image size

Commit:
A few more buttons #3

- minor button text height fix

gfursin commented:
Hmm, seems like a minor conflict with the GPU detection patch - is it possible to fix it, please? Thanks!

Commit:
A few more buttons #3

 - add dialog with correct answer request

Commit:
Merge branch 'master' into issue-3

Commit:
A few more buttons #3

 - prepare json dict with correct answer and base64 url safe encoded file to send

gfursin commented:
These things start working, but when you flip a camera, I think there is again a problem with image ratio ;) ... Don't know if it's easy to fix ...

gfursin commented:
And thanks for the update!
